README.md

```diff
@@ -1,12 +1,37 @@
-# Business Site
+# Business Site - [View Live!](https://ylvas-buiness-site.netlify.app/)
 
-Replace this readme with your own information about your project. 
+The assigment is to create a Buisness Site which has the following content:
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+* Hero image/video
+* Signup form
+* FAQ section (able to expand/collapse description)
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+### Tools and techniques
+
+* Flexbox & media queries for responsivness (mobile, tablet and desktop)
+* `<form> </form>` for Signup form
+* Accordion, an area that can collapse to show and hide text
+
+### Planning
+
+I started with a sprint planning and was breaking down the work in smaller tasks (added into Trello).
+
+In HTML file I started to add html element needed for my site, like a template.
+
+Started from the top of the site and worked down to the bottom.
+
+### Had more time, I would have done this next
+
+I feel that I once again have spent to much time on how I want the site to look like instead of just keep it simple (since design isn't my strongest side) and concentrate more on the techiques to be used to create an accordion for example.
+
+I would have wwanted to have time to play around more with the form and change the layout of it.
+
+For me it would have been so nice if I could have got some input regarding font-family, font-size, colors and images.
```

Comment:
Do you have any specific questions that I can answer regarding this? I think you've done a great work but if you feel a bit unsure, please feel free to ask here or reach out on Slack!